options/locale/locale_bg-BG.ini

```diff
@@ -767,7 +767,8 @@ pulls.merged_as=Тази заявка за сливане е била обеди
 pulls.is_closed=Тази заявка за сливане е затворена.
 pulls.has_merged=Тази заявка за сливане е обединена.
 pulls.title_wip_desc=`<a href="#">Започнете заглавието с <strong>%s</strong></a>, за да предотвратите случайно обединяване на заявката за сливане.`
-pulls.cannot_merge_work_in_progress=Тази завка за сливане е отбелязане като работа в прогрес. Премахнете <strong>%s</strong> представката от заглавието, когато е готова
+pulls.cannot_merge_work_in_progress=Тази завка за сливане е отбелязане като работа в прогрес.
```

Comment:
Please only modify the en-us language file as these other changes will be overwritten by the translation tool.

Reply:
![image](https://user-images.githubusercontent.com/2212615/107875125-63ac2600-6ebe-11eb-8dcd-34c9d6f01876.png)

Will be shown like this then in the other languages. Is that OK?

Reply:
Or would it be better to *replace* the existing pulls.cannot_merge_work_in_progress by splitting it into 2 new messages, thereby avoiding the `<strong>%s</strong>` issue seen above for the non-English languages. Instead, it would then show the English messages until they get translated.